superset/models/alerts.py

```diff
@@ -51,16 +55,16 @@ class Alert(Model):
     __tablename__ = "alerts"
 
     id = Column(Integer, primary_key=True)
-    label = Column(String(150))
+    label = Column(String(150), nullable=False)
```

Comment:
let's inherit objects that are modifiable by the user from AuditMixin:
e.g. https://github.com/apache/incubator-superset/blob/9fc37ea9f1ff032dea923012613d4e7189ada178/superset/migrations/versions/27ae655e4247_make_creator_owners.py#L65

it will require you to regenerate the migrations